posts/dialog.md

```diff
@@ -0,0 +1,12 @@
+feature: dialog
+status: use
+tags: polyfill
+kind: html
+polyfillurls:[dialog-polyfill](https://github.com/GoogleChrome/dialog-polyfill)
```

Comment:
I saw you're one of the guys behind `dialog-polyfill`. Awesome stuff! Could we maybe add a note about the browser support of the polyfill itself? What is its official browser support (of the polyfill)?